codec/unknownproto/unknown_fields.go

```diff
@@ -441,3 +441,65 @@ func (d DefaultAnyResolver) Resolve(typeURL string) (proto.Message, error) {
 	}
 	return reflect.New(mt.Elem()).Interface().(proto.Message), nil
 }
+
+// fieldTypeToProtowireType converts a descriptorpb.FieldDescriptorProto_Type to a protowire.Type.
+func fieldTypeToProtowireType(fieldType descriptorpb.FieldDescriptorProto_Type) protowire.Type {
```

Comment:
Not: as previously suggested, the name of the function could be reduced to toProtoWireType given the arguments are even fieldType